### ISSUE TITLE
Add ability to pass in both qos, partition, and walltime to remap_restarts.py

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/GEOS_Util#22]
`remap_restarts.py` currently has the ability to pass in the SLURM qos, but not partition or walltime. In doing tests at C5760 it might be nice to have these abilities.  

Maybe by default they are:
```
#SBATCH --partition=compute
#SBATCH --qos=debug
#SBATCH --walltime=1:00:00
```
unless you break the debug QOS number of tasks barrier and they go to:
```
#SBATCH --qos=allnccs
```

And then a user can specify their own of everything if desired.

I might try this myself, but I'm guessing @weiyuan-jiang can do it in minutes compared to my hours. :)

**Comment by weiyuan-jiang** [comment]
The new commit adds the time to 12 hours if the resolution is high and does not allow "debug' qos

**Comment by gmao-rreichle** [comment]
Must have been addressed with #19 or later commit.
